src/Particle/LongRange/KContainer.h

```diff
@@ -72,14 +72,14 @@ class KContainer : public QMCTraits
    * @param kc cutoff radius in the K
    * @param useSphere if true, use the |K|
    */
-  void updateKLists(const ParticleLayout& lattice, RealType kc, bool useSphere = true);
+  void updateKLists(const ParticleLayout& lattice, RealType kc, unsigned ndim, bool useSphere = true);
 
   const auto& get_kpts_cart_soa() const { return kpts_cart_soa_; }
 private:
   /** compute approximate parallelpiped that surrounds kc
    * @param lattice supercell
    */
-  void FindApproxMMax(const ParticleLayout& lattice);
+  void FindApproxMMax(const ParticleLayout& lattice, unsigned ndim);
```

Comment:
if you have the time fix the formatting of this method name since you are changing it.